server/error.go

```diff
@@ -57,31 +50,3 @@ var (
 	//GenerateServiceURL is when the service URL cannot be generated
 	ErrPrefixOrDomainMissing = errors.New("prefix or domain is missing")
 )
-
-// ErrorResponse is a standard way to return errors to the client
-//
-// Specs: http://bsvalias.org/99-01-recommendations.html
-func ErrorResponse(w http.ResponseWriter, req *http.Request, code, message string, statusCode int, log *zerolog.Logger) {
```

Comment:
By removing the method, you changed the way we respond to customers with errors. It may cause their application to crash

Reply:
It was before v1 so I thought that it will not matter but If you want I can come back to this method

Reply:
Done, adjusted ErrorResponse to gin